### DISCUSSION TITLE
Multiple Account Support?

### DISCUSSION THREAD

**Discussion by Mrw33554432** [llm-workflow-engine/llm-workflow-engine#202] [Ideas]
Probably adding a feature that allows users to sign in multiple accounts and switching between them could be a good idea.

**Comment by thehunmonkgroup** [comment]
This would need sponsorship.

**Comment by thehunmonkgroup** [comment]
https://github.com/mmabrouk/chatgpt-wrapper/releases/tag/v0.7.1 has implemented this change
